packages/common/driver-definitions/src/storage.ts

```diff
@@ -238,6 +238,11 @@ export interface IDocumentServiceEvents extends IEvent {
  */
 export interface IDocumentDeltaConnectionEvents extends IErrorEvent {
 	(event: "nack", listener: (documentId: string, message: INack[]) => void);
+	/**
+	 * @param reason - parameter will become optional in a future release to support clean disconnections.
```

Comment:
Can we also describe what this parameter is? E.g.
```suggestion
	 * @param reason - The reason for the disconnection.
	 * Note: this parameter will become optional in a future release to support clean disconnections.
```